CHANGELOG.md

```diff
@@ -45,7 +46,6 @@ release.
   outputsuffix, both, or neither for naming convention purposes. [#5162](https://github.com/DOI-USGS/ISIS3/pull/5162)
 
 ### Added
-- Added rclone to run dependencies in meta.yaml [#5183](https://github.com/DOI-USGS/ISIS3/issues/5183)
```

Comment:
@kledmundson One last thing, it looks like this entry got erroneously removed, could we add it back in?

Reply:
Done.